rs/tests/networking/BUILD.bazel

```
@@ -187,6 +187,24 @@ system_test_nns(
     ],
 )
 
+system_test_nns(
+    name = "canister_http_non_replicated_test",
+    env = {
+        "PROXY_WASM_PATH": "$(rootpath //rs/rust_canisters/proxy_canister:proxy_canister)",
+    },
+    extra_head_nns_tags = [],  # don't run the head_nns variant on nightly since it aleady runs on long_test.
+    tags = [
+        "k8s",
+        "long_test",  # since it takes longer than 5 minutes.
+    ],
```

When I tested this manually using `ict test //rs/tests/networking:canister_http_non_replicated_test` the test passed under 2 minutes so I suggest to just run this on PRs and thus drop the `long_test` tag:
```suggestion
    tags = [
        "k8s",
    ],
```

---

Done thanks